src/applications/disability-benefits/2346/components/IntroductionPage.jsx

```diff
@@ -11,6 +12,8 @@ const IntroductionPage = props => (
       prefillEnabled={props.route.formConfig.prefillEnabled}
       messages={props.route.formConfig.savedFormMessages}
       pageList={props.route.pageList}
+      verifyRequiredPrefill={props.route.formConfig.verifyRequiredPrefill}
+      unverifiedPrefillAlert={<UnverifiedPrefillAlert />}
```

Comment:
Any reason we want to render it right here instead of passing it unrendered like: `unverifiedPrefillAlert={UnverifiedPrefillAlert}`? 🙂

Reply:
This throws a console error because the parent component is expecting an element and not a function.  I will link you the dsva chat related to this.

Reply:
Thank you! 💯